contributing_to_docs/tools_and_setup.adoc

```diff
@@ -77,6 +77,20 @@ of the box, but if you are on Windows you can use http://cygwin.com/[Cygwin])
 * An editor that can strip trailing whitespace, such as
 link:https://code.visualstudio.com/[Visual Studio Code].
 
+=== Build with podman/docker
+
+1. Install _docker_ or __podman_ package
+2. From the `openshift-docs` directory, run build:
++
+----
+$ cd openshift-docs
+$ docker run --rm -it -u `id -u` -v `pwd`:/docs:Z quay.io/openshift-cs/asciibinder asciibinder build
+----
+3. Open the generated HTML file in your web browser. This will be located in the
+`openshift-docs/_preview/<distro>/<branch>` directory, with the same path and
+filename as the original `.adoc` file you edited, only it will be with the
+`.html` extension.
+
```

Comment:
I needed to perform a lot of additional steps to get this working on a Mac with podman. Thankfully @gabriel-rh was able to help me out. Should we add these steps here? Or do you think users would be able to figure it out? (I could not.)

```
=== Using Podman on a Mac

. Make a temporary directory for Ruby in your home directory for when it runs in the container by using the following commands:
+
----
$ cd
$ mkdir ruby-tmp
$ chmod 1777 ruby-tmp
----
. Initialize the Podman machine for your home directory. You must set the security model to none so that symbolic links work.
+
----
$ podman machine init -v /Users/<username>/:/mnt/top:security_model=none
----
. Start the machine by running the following command:
+
----
$ podman machine start
----
. Connect to the machine by using ssh:
+
----
$ podman machine ssh
----
. Change to the home directory and ensure that the `ruby-tmp` directory is sticky by using the following commands:
+
----
$ pwd
$ cd /mnt/top
$ ls -al ruby-tmp/
----
. Change to your home directory and run the following command:
+
----
$ podman run --rm -it -v /mnt/top/ruby-tmp:/tmp -v `pwd`:/docs quay.io/openshift-cs/asciibinder asciibinder build --distro <distribution name><1>
----
<1> Name of the distribution, for example, `openshift-enterprise` or `openshift-acs`
. Exit the vm when the job is completed. The rendered HTML files are located in the `_preview` directory.

```

Reply:
I'm not using mac so I can't verify it :(

Reply:
I asked @mtodor to check it and it looks like docker is working fine without any magic. How about removing `Podman` and having it only for docker? I thought that podman is a drop-in replacement but apparently not.